### DISCUSSION TITLE
Error reported by Crashlytics in Huawei with Android 11

### DISCUSSION THREAD

**Discussion by Gustav-Papinotas** [invertase/react-native-firebase#6155] [Q&A]
Hi!!

I got an error reported by crashlytics...

![image](https://user-images.githubusercontent.com/87195390/160001530-579905d0-1171-47a6-9850-5ee81599759f.png)

It happens on Huawei devices with android 11.
Apparently the error occurs in the in-app-messaging module when my app start.

Has something like this happened to someone? or know if it has been corrected in any release. I currently work with firebase modules in version 12.5.0

**Comment by mikehardy** [comment]
Firebase Crashlytics is the reason you know about this crash, but I do not see anything related to firebase or react-native-firebase in the stack frames. So this does not look to me like it actually is from firebase or anything firebase-related and I don't think it is actionable here.

Unrelated, but 12.5.x versions are old, we're on 14.7.0 now and the upgrade should be pretty easy, you should stay up to date if possible, versioning philosophy here https://invertase.io/blog/react-native-firebase-versioning and the CHANGELOG with upgrade instructions here, noted on each major version (they're usually tiny...) https://github.com/invertase/react-native-firebase/blob/main/CHANGELOG.md

**Reply by Gustav-Papinotas** [reply]
Thanks a lot, i try upg the library!
